rcl_executor/include/rcl_executor/handle.h

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2018 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/micro-ROS/rcl_executor.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#ifndef RCL_EXECUTOR__HANDLE_H_
+#define RCL_EXECUTOR__HANDLE_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <rcl/rcl.h>
+
+/// TODO (jst3si) Where is this defined? - in my build environment this variable is not set.
+#define ROS_PACKAGE_NAME "rcl_executor"
+
+/// Enumeration for timer, subscription, guard conditions etc to be waited on.
+typedef enum
+{
+  SUBSCRIPTION,
+  GUARD_CONDITION,
```

Comment:
services, clients, guard conditions are not supported yet, these still need to be declared to reserve numbers?

Reply:
I already included them in the enum.

Reply:
Resolved in 256b4bbd410f753b51c792370dc3d0ae33f1618f